extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -122,4 +128,11 @@ void build(CombinedIndexBuildItem index,
         }
     }
 
+    private boolean skipRepository(ClassInfo classInfo) {
```

Comment:
Maybe rename to `isNotLeafType` and put it in the common processor ?

Reply:
I can pull it up, but I wanted the name to be clear that this is only for repositories.

Reply:
moved up.